code/espurna/config/hardware.h

```diff
@@ -3262,17 +3262,31 @@
     // Buttons
     #define BUTTON1_PIN                 4
     #define BUTTON1_CONFIG              BUTTON_PUSHBUTTON | BUTTON_DEFAULT_HIGH
+    #define BUTTON1_RELAY               1
+    // the defaults are reasonable, but you can change them as desired
+    //#define BUTTON1_PRESS               BUTTON_ACTION_NONE
+    //#define BUTTON1_CLICK               BUTTON_ACTION_TOGGLE
+    //#define BUTTON1_DBLCLICK            BUTTON_ACTION_AP
+    //#define BUTTON1_LNGCLICK            BUTTON_ACTION_RESET
+    //#define BUTTON1_LNGLNGCLICK         BUTTON_ACTION_FACTORY
 
     // Relays
     #define RELAY1_PIN                  14
     #define RELAY1_TYPE                 RELAY_TYPE_NORMAL
 
     // LEDs
+
+    // LED1 (red) indicates wifi activity
     #define LED1_PIN                    12
     #define LED1_PIN_INVERSE            1
+    #define LED1_MODE                   LED_MODE_WIFI
 
+    // LED2 (blue) indicates on/off state; you could use LED_MODE_FOLLOW_INVERSE
+    // so that the LED lights the button when 'off' so it can be found easily.
```

Comment:
Note of the FINDME mode, SHPX as an example.
I have not tracked color scheme all the way through all of the devices, but comparing to SPHX again - red is relay state, blue is wifi; coloring is other way around.

Reply:
OK! I reverse the red/ blue roles.